RELEASE.md

```diff
@@ -0,0 +1,22 @@
+# Release Process
+
+- leave the Cargo.toml package version alone :)
+- trigger the `tag and publish crate [manual]` action and specify the release increment (major, minor, or patch)
+  - once the action completes, the Cargo.toml will be updated and the crate will be live on crates.io
+- create a release (with title, notes, thanks, etc) and tie it to the tag that was created by the action / new crate version
```

Comment:
could this also be handled in the action? or maybe having the github release be manual is a good human checkpoint

Reply:
yeah I was thinking exactly that the github release is a good human checkpoint, nice to give a shout out to new contributors or provide links or whatev else, also easy to backfill if we forget it or something too. tho, it might be that that kinda stuff could be added after the fact 🤔 lets see how it goes having it be the human checkpoint 😄